client/task-list/task-list.js

```diff
@@ -349,15 +345,9 @@ export const TaskList = ( {
 								title={ task.title }
 								completed={ task.completed }
 								content={ task.content }
-								onClick={
-									! expandingItems || task.completed
-										? task.onClick
-										: () => setCurrentTask( task.key )
-								}
-								expandable={ expandingItems }
-								expanded={
-									expandingItems && currentTask === task.key
-								}
+								expandable={ expandingItems && task.expandable }
```

Comment:
@octaedro Does this need to be added to the new task list as well?  This code will be removed soon.